ocaml/testsuite/tests/typing-layouts-err-msg/value.ml

```diff
@@ -2,7 +2,6 @@
    flags = "-extension layouts_alpha"
    * expect
 *)
-(* CR layouts v2.9: all error messages below here are not reviewed for quality *)
```

Comment:
Nits regarding the last line of the error for the example beginning on line 306:
```
let f (x : t_float64) = `A x;;
[%%expect{|
Line 1, characters 27-28:
1 | let f (x : t_float64) = `A x;;
                               ^
Error: This expression has type t_float64
       but an expression was expected of type ('a : value)
       The layout of t_float64 is float64, because
         of the definition of t_float64 at line 5, characters 0-24.
       But the layout of t_float64 must be a sublayout of value, because
         it's a field of a polymorphic variant.
|}];;
```
Polymorphic variants have at most one argument (unlike normal variants), so I would say "the field" rather than "a field".  Also, maybe better to say "the type of the field" than "the field"?

Reply:
fixed